keyboard-racing/src/components/Home/Home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Home extends React.Component {
+
+  componentDidMount(){
+    const urlParams = new URLSearchParams(window.location.search);
+    const myParam = urlParams.get('code');
+    if(myParam){
+      localStorage.setItem('code', myParam);
+      window.location.href = '/#/';
```

Comment:
Should we store Url's in an a constant object instead of hardcoding?

Reply:
Kind of not sure because that won’t ever change. Can you send me a link of a good way to do it? Similar to the config.js?

Reply:
Something like this is what I'm used to

![ex](https://user-images.githubusercontent.com/22139388/73602108-07d56000-453c-11ea-85c4-f5f0e18f2c87.PNG)